types/PayloadErrors.ts

```diff
@@ -0,0 +1,17 @@
+export class PayloadError extends Error {
+  constructor(message: (typeof payloadErrors)[number]) {
+    super(message);
+    this.name = "PayloadError";
+  }
+}
+
+const payloadErrors = [
+  "playerNotFound",
+  "targetNotFound",
+  "playerAlreadyExists",
+  "gameAlreadyStarted",
+  "playerIsDead",
+  "targetIsDead",
+  "youCannotVoteNow",
+  "playerIsAlreadyConnected",
```

Comment:
```suggestion
  "playerIsAlreadyConnected",
  "playerCannotBeReady",
```

See suggested changes to server/src/engine/Game.ts